Subject: [PATCH] dhcpc: fix -Wextra implicit-fallthrough

Same as d6_dhcpc: canonicalize "fall right through" comments,
add the missing marker before default:.
diff --git a/networking/udhcp/dhcpc.c b/networking/udhcp/dhcpc.c
index e0309d2..874e14b 100644
--- a/networking/udhcp/dhcpc.c
+++ b/networking/udhcp/dhcpc.c
@@ -1501,7 +1501,7 @@ int udhcpc_main(int argc UNUSED_PARAM, char **argv)
 			got_SIGUSR1:
 				log1s("entering renew state");
 				change_listen_mode(LISTEN_KERNEL);
-				/* fall right through */
+				/* fall through */
 			case RENEW_REQUESTED: /* in manual (SIGUSR1) renew */
 			case RENEWING:
 				if (packet_num == 0) {
@@ -1537,7 +1537,7 @@ int udhcpc_main(int argc UNUSED_PARAM, char **argv)
 				/* Switch to bcast receive */
 				change_listen_mode(LISTEN_RAW);
 				packet_num = 0;
-				/* fall right through */
+				/* fall through */
 			case REBINDING:
 				/* Lease is *really* about to run out,
 				 * try to find DHCP server using broadcast */
@@ -1594,6 +1594,7 @@ int udhcpc_main(int argc UNUSED_PARAM, char **argv)
 				/* Two SIGUSR1 received, start things over */
 				change_listen_mode(LISTEN_NONE);
 				d4_run_script_deconfig();
+				/* fall through */
 
 			default:
 			/* case RELEASED: */
